lib/utils.js

```diff
@@ -127,7 +127,14 @@ const config = values => {
   return result;
 };
 
-// Structures input from form
+/*
+ * Formats raw data from form to object
+ *
+ * @param {object} body - Raw data object
```

Comment:
this doesn't actually help much. Please use the [`typedef` descriptor](http://usejsdoc.org/tags-typedef.html) to break down what is in both the param and returns.

[Here is an example usage in our code](https://github.com/punchcard-cms/punchcard/blob/64921680ac6bd635725bc09908ea6a292dded73c/lib/utils.js#L63)

```
/*
 * @typedef FormattedDate
 * @type object
 *
 * @property {string} date - The date, formatted YYYY-MM-DD (e.g. 2016-05-25)
 * @property {string} time - The time, formatted HH:mm (e.g. 13:01)
 * @property {string} zone - The time zone, formatted Z (e.g. America/New_York)
 */

/*
 * Formats ISO date in to requisite components
 *
 * @param {string} date - ISO UTC formatted date
 * @param {string} zone - Timezone to retrieve date in
 *
 * @returns {FormattedDate} - The individually formatted date components
 */
```